### PR TITLE
chore(demo): fix demo update nx configs to project.json

### PR DESCRIPTION
**Pull request by sevaru**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [x] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Demo and docs app not working

## What is the new behavior?

Demo and docs app fixed 🤖

### CODE REVIEW

**Comment by splincode** [comment]
@sevaru please, use `main` branch for target

File: apps/remote/src/app/remote-module/remote-module.component.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:host {
+  display: block;
+  background-color: beige;
```
Comment (MarsiBarsi) [inline apps/remote/src/app/remote-module/remote-module.component.css:3]: do all these text colors look good? should not we use some colors by Taiga better https://taiga-ui.dev/colors ?

**Review by MarsiBarsi — COMMENTED** [review]
I see that there are some linting issues as well. Let's check later that eslint and prettier are here and feel good (and fix if they are not) + run all linters with `--fix` and make it all clean

image

**Comment by sevaru** [comment]
@MarsiBarsi I'll address linting issues in a separate PR to streamline the process.

**Review by MarsiBarsi — DISMISSED** [review]

**Review by MarsiBarsi — APPROVED** [review]

**Pull request merged**